1.9/monitoring/logging/logging-api-examples.md

```diff
@@ -0,0 +1,86 @@
+---
+post_title: Logging API Examples
+feature_maturity: preview
+menu_order: 4
+---
+
+The following examples exercise the Logging API using [Bash](https://www.gnu.org/software/bash/), [Curl](https://curl.haxx.se/), and [jq](https://stedolan.github.io/jq/).
```

Comment:
`This topic provides common usage examples for the logging API. They examples use [Bash](https://www.gnu.org/software/bash/), [Curl](https://curl.haxx.se/), and [jq](https://stedolan.github.io/jq/).`